### PR TITLE
done

### PR DESCRIPTION
**Pull request by darmhoo**

### CODE REVIEW

File: src/components/PaymentLog.vue [hunk 1]
```diff
@@ -570,6 +573,9 @@ export default {
                     this.apiUrls.businessTypes
                 );
                 this.businessTypes = fetchBusinessTypes.data.data.data;
+                this.businessTypes = this.businessTypes.filter(item => {
+                  return item.name === "Altara Credit Cash Loan" || item.name === "Altara Credit Products"
```
Comment (AltaraNg) [inline src/components/PaymentLog.vue:577]: This is not the right way to do it, filter by if name has Altara Credit

File: src/components/PaymentLog.vue [hunk 2]
```diff
@@ -352,6 +353,8 @@ export default {
             this.salesLogForm.repayment_cycle_id.name === "custom"
                 ? (this.customDateToggle = true)
                 : (this.customDateToggle = false);
+
+            this.getCalc();
```
Comment (AltaraNg) [inline src/components/PaymentLog.vue:357]: why is this here???

Comment (darmhoo) [reply src/components/PaymentLog.vue:357]: that is the trigger for payment cycle

File: src/components/PaymentLog.vue (hunk 1 above)
Comment (darmhoo) [reply src/components/PaymentLog.vue:577]: okk

**Review by AltaraNg — APPROVED** [review]
